<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WdgStrokeSelection</class>
 <widget class="QWidget" name="WdgStrokeSelection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>869</width>
    <height>601</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>New Image</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" rowspan="2">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Stroke</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="1">
          <widget class="QComboBox" name="typeBox">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <item>
            <property name="text">
             <string>Current Brush</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Line selection</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Fill:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Type:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QSpinBox" name="lineSize">
           <property name="suffix">
            <string/>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>1000000</number>
           </property>
           <property name="value">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="lblSize">
           <property name="text">
            <string>Width:</string>
           </property>
           <property name="buddy">
            <cstring>intSize</cstring>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QComboBox" name="sizeBox">
           <item>
            <property name="text">
             <string>px</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>mm</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>inch</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QComboBox" name="fillBox">
           <item>
            <property name="text">
             <string>None</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Paint color</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Background color</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Custom color</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Foreground color</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Line:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="KColorButton" name="colorSelector">
           <property name="text">
            <string>Color</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
           <property name="default">
            <bool>false</bool>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QComboBox" name="lineColorBox">
           <item>
            <property name="text">
             <string>Foreground color</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Background color</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Custom color</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="5" column="2">
          <widget class="KColorButton" name="colorFillSelector">
           <property name="text">
            <string>Color</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KColorButton</class>
   <extends>QPushButton</extends>
   <header>kcolorbutton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
